hypervisor/lib/string.c

```diff
@@ -395,3 +395,54 @@ int strncmp(const char *s1, const char *s2, size_t n)
 
 	return *s1 - *s2;
 }
+
+/*
```

Comment:
can you please update function comments like above? e,.g.  the function strncpy_s()